erpnext/patches/v11_0/rename_employee_loan_to_loan.py

```diff
@@ -9,9 +9,9 @@ def execute():
 	if frappe.db.table_exists("Employee Loan") and not frappe.db.table_exists("Loan"):
 		frappe.rename_doc("DocType", "Employee Loan", "Loan", force=True)
 
-	frappe.reload_doc("hr", "doctype", "loan_application")
-	frappe.reload_doc("hr", "doctype", "loan")
-	frappe.reload_doc("hr", "doctype", "salary_slip_loan")
+	frappe.reload_doc("loan_management", "doctype", "loan_application")
+	frappe.reload_doc("loan_management", "doctype", "loan")
```

Comment:
If we are not doing patch for existing loan to new loan, then this patch also does not make any sense.